src/main/scala/cromwell/engine/backend/local/LocalFileSystem.scala

```diff
@@ -113,14 +124,35 @@ trait LocalFileSystemOperations {
   private def stageWdlFile(wdlFile: WdlFile, hostInputsPath: Path): WdlFile = {
     val originalPath = Paths.get(wdlFile.value)
     val executionPath = hostInputsPath.resolve(originalPath.getFileName.toString)
+
+    val localizers = LocalizationStrategies map {
+      case "hard-link" => localizePathViaHardLink _
+      case "soft-link" => localizePathViaSymbolicLink _
+      case "copy" => localizePathViaCopy _
+    }
+
+    val attemptedLocalization = Stream(localizers: _*) map { _(originalPath, executionPath) } find { _.isSuccess }
+    attemptedLocalization match {
+      case None => throw new UnsupportedOperationException(s"Could not localize $wdlFile -> $executionPath")
+      case _ =>
+    }
```

Comment:
Yeah... I'll look into better error handling.  Prior to this we were calling some Java APIs that threw exceptions so at least my solution is no worse, though it can be better.